LICENSE

```diff
@@ -252,3 +252,28 @@ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
 THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+-------------------------------------------------------------------------------
+
+# https://github.com/microsoft/vscode-icons
+    MIT License
```

Comment:
Sorry for bothering but please remove indentation :D

Reply:
This indent is came from vscode-icon's LICENSE file. Should I remove it?

Reply:
The whole statement is same so removing indent seems better, but not "must" :)
I will reorganize this with other things so let's go as it is! :D